### PR TITLE
Make rspec generate HTML coverage reports

### PR DESCRIPTION
**Pull request by jhbrooks**

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -2,13 +2,16 @@
...
+
+SimpleCov.formatters = [
+  Codacy::Formatter,
+  SimpleCov::Formatter::HTMLFormatter
+]
+
 SimpleCov.start 'rails' do
   add_group "Calculators", "app/calculators"
   add_group "File Handlers", "app/file_handlers"
-  add_group "Jobs", "app/jobs"
```
Comment (mathmerized) [inline spec/rails_helper.rb]: Why did you remove jobs?

Comment (jhbrooks) [inline spec/rails_helper.rb]: Jobs is one of the default groups, so we don't need to add it.
